static/openapi/hss-spec.json

```diff
@@ -0,0 +1,3701 @@
+                            },
+                            {
+                                "$ref": "#/components/schemas/PqQuantization"
+                            },
+                            {
+                                "$ref": "#/components/schemas/SqQuantization"
+                            }
+                        ],
+                        "title": "Quantization Config",
+                        "description": "Configure the precision of the numbers used to represent model parameters or embeddings. This technique maps continuous or high-precision values to smaller discrete values, effectively compressing the embeddings' size and reducing the memory and computational requirements for storing and processing them.",
+                        "discriminator": {
+                            "propertyName": "type",
+                            "mapping": {
+                                "none": "#/components/schemas/NoneQuantization",
+                                "pq": "#/components/schemas/PqQuantization",
+                                "sq": "#/components/schemas/SqQuantization"
+                            }
+                        }
+                    }
+                },
+                "type": "object",
+                "required": [
+                    "name",
+                    "model_name",
+                    "index_config",
+                    "quantization_config"
+                ],
+                "title": "SemanticSearchIndex"
+            },
+            "SortOrderType": {
+                "type": "string",
+                "enum": [
+                    "asc",
+                    "desc"
+                ],
+                "title": "SortOrderType",
+                "description": "Sort order."
+            },
+            "SqQuantization": {
+                "properties": {
+                    "type": {
+                        "type": "string",
+                        "enum": [
+                            "sq"
+                        ],
+                        "const": "sq",
+                        "title": "Type",
+                        "description": "Type of quantization.",
+                        "default": "sq"
+                    },
+                    "bits": {
+                        "type": "integer",
+                        "title": "Bits",
+                        "description": "The number of output bits after quantization.",
+                        "examples": [
+                            8
+                        ]
+                    }
+                },
+                "type": "object",
+                "required": [
+                    "bits"
+                ],
+                "title": "SqQuantization",
+                "description": "In Scalar quantization (SC), each value is quantized independently, which is straightforward but may\nnot capture the correlations between the embeddings' different dimensions."
```

Comment:
```suggestion
                "description": "In Scalar quantization (SC), each value is quantized independently, which is straightforward but might\nnot capture the correlations between the embeddings' different dimensions."
```

Reply:
Do not capitalize the quantization types. 

scalar quantization
planar quantization
etc.